tests/requirements-test.txt

```diff
@@ -1,5 +1,6 @@
 versioneer
-pytest
+# pytest v3.0 (from 19.08.2016) is no longer compatible with Python 3.2.x
+pytest<3.0 ; python_version >= '3.2' and python_version < '3.3'
```

Comment:
I prefer to have two lines with pytest specification, see:
https://github.com/DataMedSci/pymchelper/blob/master/tests/requirements-test.txt

Your version works because:
 - for python 3.2 `pytest<3.0`
 - for python > 3.2 pytest itself won't be found in requirements-test.txt (as there is no corresponding line to it). But why it is passing tests ? There are pytest-* plugins listed, which require pytest itself. So in that case pytest will be fetched as a requirement of them.

According to Python ZEN: `Explicit is better than implicit.` I think it is better to explicitely specify what to use for each python version